tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -103,10 +103,30 @@ def get_valid_emails_from_list(self, targets):
         return emails
 
     def get_event_owner_email(self, targets, event):
-        if 'event-owner' in targets and self.config.get('ldap_uri', False):
+        self.logger.debug('getting owner email')
```

Comment:
this probably too verbose for the common case without providing much informational value.